exasol_advanced_analytics_framework/interface/create_event_handler_udf.py

```diff
@@ -1,10 +1,44 @@
+from exasol_advanced_analytics_framework.event_handler.event_handler_base import \
+    EventHandlerBase
+from exasol_advanced_analytics_framework.event_handler.event_handler_context import EventHandlerContext
+import importlib
+
+from exasol_advanced_analytics_framework.event_handler.event_handler_result import \
+    EventHandlerResult
+
 
 class CreateEventHandlerUDF:
     def __init__(self, exa):
         self.exa = exa
 
     def run(self, ctx) -> None:
-        ctx.emit("return query")
-        ctx.emit("status")
-        ctx.emit("query 1")
-        ctx.emit("query 2")
+        parameters = ctx.parameters
+        bucketfs_connection = ctx.bucketfs_connection
+
+        event__handler_context = EventHandlerContext(bucketfs_connection)
```

Comment:
you need to handle here two cases, init (event handler doesn't exists, yet ), handle event (event handler exists, but needs to be deserialized)

Reply:
the method that is invoked just below (_get_event_handler_obj) actually does what you say.

Reply:
Ah ok, the reason why I didn't recognize it, was that you also need to store and load the context it keeps the information about temporary objects

Reply:
My idea was, you have one object which keeps event handler and context as attributes, and you load and store this object

Reply:
I had separated them. It's also a good idea to keep it together.